qutebrowser/misc/tabsfinder.py

```diff
@@ -0,0 +1,43 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+import re
+from collections import namedtuple
+
+from qutebrowser.utils import objreg
+
+Tab = namedtuple('Tab', 'win_id index url title')
+
+
+def all_tabs(skip_win_id=None):
+    for win_id in objreg.window_registry:
+        tabbed_browser = objreg.get('tabbed-browser', scope='window', window=win_id)
+
+        if skip_win_id is not None and win_id == skip_win_id:
```

Comment:
Since `win_id` will not be `None` anyways, you can drop the `if skip_win_id is not None` part.